### DISCUSSION TITLE
全球首顆「可以吃的NFT」在台灣！

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#248] [科技新知]
「共創者之夜」於12月21日舉辦，將米其林餐廳RAW化身成為藝術展演的場地，讓八位共創者體驗多層次的視覺、聽覺、嗅覺、味覺及觸覺的藝術堆疊。透過張逸軍老師與舞者們曼妙柔軟的舞蹈，再進入江振誠主廚的故事特色料理，接著著踏入黃心健導演的 VR 元宇宙世界，最後回到真實世界品嚐甜點。
江振誠主廚表示：「料理是種行為藝術，透過此次跨界合作延伸八角哲學，讓記憶不再只是記憶，而是通往未來世界的鑰匙。這次的跨界合作與黃心健導演、張逸軍老師一起將可以吃的NFT實現跟完成，也是我一直在嘗試與突破的。希望大家能夠透過共創者之夜參與藝術的多樣性、創造性及延續性。」

[https://www.marieclaire.com.tw/lifestyle/taste/62827/raw-nft-2022?atcr=8c4b3](url)

**Comment by Joan0518** [comment]
NFT結合不同產業的例子真的越來越多了~~

**Comment by chupochun** [comment]
原來那麼多人都在討論NFT，但是我居然是最近才知道的，我真是鄉巴佬
